src/sdk/rest/API.js

```diff
@@ -1311,6 +1311,113 @@ OWT_REST.API = (function(OWT_REST) {
+     * * @callback onAnalyticsList
+     * * @param {Array.<{id: string, analytics: Object, media: Object}>} analyticsList            -The analytics list.
+     * * @param {Object} analyticsList[x].analytics            -The information of the analytics.
+     * * @param {string} analyticsList[x].analytics.algorithm  -The algorithm of the analytics.
+     * * @param {Object} analyticsList[x].media                -The media description of the analytics, which must follow the definition of object "MediaSubOptions" in section "3.3.11 Participant Starts a Subscription" in "Client-Portal Protocol.md" doc.
+  */
+  /**
+     * @function getAnalytics
+     * @desc This function gets the all the ongoing analytics in the specified room.
+     * @memberOf OWT_REST.API
+     * @param {string} room                          -Room ID.
+     * @param {function} callback                    -Callback function on success
+     * @param {function} callbackError               -Callback function on error
+     * @example
+  var roomId = '51c10d86909ad1f939000001';
+  OWT_REST.API.getAnalytics(roomId, function(analyticsList) {
+    console.log('Analytics:', analyticsList);
+  }, function(status, error) {
+    // HTTP status and error
+    console.log(status, error);
+  });
+     */
+  var getAnalytics = function(room, callback, callbackError) {
+    send('GET', 'rooms/' + room + '/analytics/', undefined, function(analyticsList) {
+      var result = JSON.parse(analyticsList);
+      callback(result);
+    }, callbackError);
+  };
+
+  /*
+     * * @callback onStartingAnalyticsOK
+     * * @param {Object} analyticsInfo               -The object containing the information of the server-side analytics.
+     * * @param {string} analyticsInfo.id            -The analytics ID.
+     * * @param {Object} analyticsInfo.analytics     -The information of the analytics.
+     * * @param {string} analyticsInfo.analytics.algorithm  -The algorithm of the analytics.
+     * * @param {Object} analyticsInfo.media         -The media description of the analytics, which must follow the definition of object "MediaSubOptions" in section "3.3.11 Participant Starts a Subscription" in "Client-Portal Protocol.md" doc.
+  */
+  /**
+     * @function startAnalytics
+     * @desc This function starts a analytics in the specified room.
+     * @memberOf OWT_REST.API
+     * @param {string} room                          -Room ID.
+     * @param {string} algorithm                     -The algorithm ID.
+     * @param {Object} media                         -The media description of the analytics, which must follow the definition of object "MediaSubOptions" in section "3.3.11 Participant Starts a Subscription" in "Client-Portal Protocol.md" doc.
+     * @param {onStartingAnalyticsOK} callback       -Callback function on success
+     * @param {function} callbackError               -Callback function on error
+     * @example
+  var roomId = '51c10d86909ad1f939000001';
+  var algorithm = 'testalgo123';
```

Comment:
I think we should clarify on the algorithm ID here in API doc.  testalog123 will be confusing. Should clarify it is the guid in plugin.cfg.